### ISSUE TITLE
modern dock

### ISSUE THREAD

**Issue by lacrimos** [AnemoneTeam/Anemone#145]
I have a modern dock but when I apply in anemone it didn't work

**Comment by Alfroggy** [comment]
Hi.
1) Do you have a ModernDockMask in a 'com.apple.springboard' folder in your 'Bundles' folder ?
2) A ModernDockOverlay too ?
3) What size dock image(s) do have ?
4) Do you have an 'Info.plist' file in the root of your active theme ?
4 b) If so, is there a dock-relative entry in it ?

**Comment by coolstar** [comment]
Can you try installing "ClassicDock" from the BigBoss repo and enabling only the "ModernDock" option under ClassicDock?

**Comment by lacrimos** [comment]
1.yes
2.yes
3. ModernDockMask@3x W 750 H 97.0
ModernDockMask-667h@2x.png the same
ModernDockOverlay-667h@2x.png the same
I have a info.plist in the dock folder
4. I don't know
ModernDockOverlay-736h@3x.png the same size all

**Comment by lacrimos** [comment]
I have installed Classic Dock yesterday still don't work

**Comment by lacrimos** [comment]
I'm on iPhone 6+

**Comment by coolstar** [comment]
Can you try disabling all themes and only enable the "Modern Dock" option in the classicdock theme? make sure the "classicdock" option is not enabled.

**Comment by Alfroggy** [comment]
Ok, 6+ needs a -736h@3x Mask too I reckon.
a) is that Info.plist is in the root of your dock theme, as in /Themes/MyDock.theme/Info.plist ?
b) is there a 'DockMode' key in that plist ?

**Comment by lacrimos** [comment]
I have reinstalled classicdock once again disabled all themes activate modern dock still don't work I have checked the info.plist and only package name > dock thats all

**Comment by lacrimos** [comment]
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
  <key>PackageName</key>
  <string>Docks</string>
</dict>
</plist>

**Comment by Alfroggy** [comment]
Ok. Yeah, ModernDock mode asks for no DockMode entry in the Info.plist so that's not your issue.

You don't have a -736h@3x mask at all then ?

**Comment by lacrimos** [comment]
ModernDockMask-736h@3x.png yes I have 
so without thst info.plist it will not work?

**Comment by Alfroggy** [comment]
Cool. Then I can't say why it doesn't work. Are you in zoomed mode in Settings>Lighting and display settings ?
No need for an Info.plist (note the capitalisation, it's all about details in iOS theming) in your case, so that's not what to focus on now.

**Comment by lacrimos** [comment]
I'm in normal mode

it's possible that a tweak cause that issue?

**Comment by Alfroggy** [comment]
Yes. Check your themes for anything affecting the dock.

It perhaps is the transparency setting in Settings>General>Accessibility Settings>Contrast>Reduce Transparency

**Comment by coolstar** [comment]
Anemone can theme the dock properly even if Reduce Transparency is enabled

**Comment by Alfroggy** [comment]
It used not to be the case. What Anemone version are you on @lacrimos ?

**Comment by lacrimos** [comment]
on the latest version update is morning

**Comment by lacrimos** [comment]
reduce transparency is on

**Comment by Alfroggy** [comment]
Yeah you want it off for the underlay to show through the mask. Transparency off turns the mask opaque.

**Comment by lacrimos** [comment]
I have restored my phone only to see if a tweak cause rhis problem but still not work... do you mean transparency in settings?
so another question is which size a dock  need to have  for 6+  or 6 because coolstar  never tells which size the dock need to have

**Comment by Alfroggy** [comment]
Yes Reduced Transparency in Accessibility Settings needs to be kept OFF for your dock to show through the mask.

-667@2x has to be 750px in width for the 6/6s. Mine's height is 192px and shows untouched on springboard: no shrink/squash/cut or whatever distortion. 

Coolstar is aware of the lacking image dimensions. He'll update the wiki in time, has been very busy squashing the last few bugs and smoothing the experience.

**Comment by lacrimos** [comment]
yes is off

and for 6+? do you know the size

**Comment by lacrimos** [comment]
ok I have a working dock  but in this folder there are 12 files in the one I try to apply there are only 4 

ModernDockMask-667h@2x.png
ModernDockMask-736h@2x.png
ModernDockMask@2x.png
ModernDockMask-736h@3x.png
ModernDockMask@3x.png
SBDockBG-667h@2x~iphone.png
SBDockBG-736h@2x~iphone.png
SBDockBG-736h@3x~iphone.png
SBDockBG@2x~iphone.png
SBDockReflectionHeight@2x.png
SBDockBG@3x~iphone.png
SBDockReflectionHeight@3x.png
